lib/src/base-styles/_nx-page.scss

```diff
@@ -11,8 +11,10 @@
   display: grid;
   grid-template-columns: auto 1fr auto;
   grid-template-areas:
-    "title tags actions"
-    "description description description";
+    "title sub-title actions"
+    "description description description"
+    "tags tags tags";
+  column-gap: $nx-spacing-md;
```

Comment:
It appears that when the subtitle is omitted, the column gaps on either side of that grid cell double up: 
![image](https://user-images.githubusercontent.com/3630091/116090059-7a66ba00-a671-11eb-8760-ce02af38f18b.png)

Reply:
> It appears that when the subtitle is omitted, the column gaps on either side of that grid cell double up:
> ![image](https://user-images.githubusercontent.com/3630091/116090059-7a66ba00-a671-11eb-8760-ce02af38f18b.png)

I noticed that but I cannot find a solution. I didn't think that it was worth worrying about.

Reply:
> I noticed that but I cannot find a solution. I didn't think that it was worth worrying about.

Howe about using margins instead of gap?  Luckily this is the only row of the grid that has multiple items should it should be pretty straightforward

Reply:
> It appears that when the subtitle is omitted, the column gaps on either side of that grid cell double up:
> ![image](https://user-images.githubusercontent.com/3630091/116090059-7a66ba00-a671-11eb-8760-ce02af38f18b.png)

Margin works, and I can't remember why I switched that out for `column-gap`: https://github.com/sonatype/sonatype-react-shared-components/pull/363/commits/b4660408e77ea0f60e58dc9f88587b385780dbd3